utils/cliutils/commandsflags.go

```diff
@@ -624,6 +626,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  syncDeletes,
 		Usage: "[Optional] Specific path in Artifactory, under which to sync artifacts after the upload. After the upload, this path will include only the artifacts uploaded during this upload operation. The other files under this path will be deleted.` `",
 	},
+	uploadArchive: cli.StringFlag{
+		Name:  archive,
+		Usage: "[Optional] Set to \"zip\" to deploy to Artifactory in a ZIP archive.` `",
```

Comment:
...to deploy to Artifactory in a ZIP archive. --> ...to deploy to the files to Artifactory in a ZIP archive.